src/app/components/project-selector/projectSelector.html

```diff
@@ -5,14 +5,18 @@
     <md-icon md-font-icon="fa fa-angle-down fa-sm"></md-icon>
   </md-button>
   <md-menu-content width="4" class="md-menu-bar-menu md-dense">
+    <md-menu-item>
+      <p ng-show="!$ctrl.isAuthenticated" align="center">
+        <md-button ng-click="$ctrl.signIn()">Sign In</md-button>to see your projects
+      </p>
+    </md-menu-item>
```

Comment:
so this leaves a bunch of weird space at the top of the menu when the user is logged in.

Possibly a better solution would be to just add a small button `Sign In` with a tooltip that tells the says something like "Sign in to see your projects", and put it in the same spot where `<md-checkbox` is on line 16. If authenticated, show the checkbox, otherwise show the sign in button